shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -43,7 +43,7 @@
  */
 public final class WireCommands {
     public static final int WIRE_VERSION = 1;
```

Comment:
As per PDP-21, I believe we need to increment the wire version, since we are adding new commands.

https://github.com/pravega/pravega/wiki/PDP-21:-Protocol-revisioning

Reply:
fixed. It is now 2